clients/src/test/java/org/apache/kafka/common/requests/DescribeAclsResponseTest.java

```diff
@@ -53,37 +54,37 @@
 
     @Test(expected = UnsupportedVersionException.class)
     public void shouldThrowOnV0IfNotLiteral() {
-        new DescribeAclsResponse(10, ApiError.NONE, aclBindings(PREFIXED_ACL1)).toStruct(V0);
+        DescribeAclsResponse.prepareResponse(10, ApiError.NONE, aclBindings(PREFIXED_ACL1)).toStruct(V0);
```

Comment:
I think we should convert these tests not to rely on `AclBinding` unless they are testing `prepareResponse` specifically.